### ISSUE TITLE
Dropbox changed API

### ISSUE THREAD

**Issue by MilosStanic** [mplodowski/backupmanager-plugin#1]
Hello, 

I'm using your plugin to backup to DropBox, and a few weeks ago I started getting message: 
```
Dropbox\Exception_BadRequest: HTTP status 400
{"error": "v1_retired"} in /var/www/provedk/october/plugins/renatio/backupmanager/vendor/dropbox/dropbox-sdk/lib/Dropbox/RequestUtil.php:250
```

Please resolve this issue. 

Thanks

**Comment by mplodowski** [comment]
This issue was resolved some time ago.

Please update plugin to latest version and follow documentation.

https://github.com/mplodowski/backupmanager-plugin#dropbox-configuration

**Comment by MilosStanic** [comment]
Sorry, I was on build 419 and it didn't offer to update Backup Manager. 
Now it offers to update all. 

So now it's necessary to obtain a new set of keys from DropBox?
Is it necessary to define DropBox filesystem? Because I want to keep my uploads and media files on my server. 

I don't know when I'll upgrade to latest build of October because I have about 20k users, and I don't know if it will affect the stability of the system. 

Thanks.

**Comment by mplodowski** [comment]
October released version with Laravel 5.5 lately as stable.

I would recommend you to update to latest build of October and then update Backup Manager as soon as possible.

Go to your app on Dropbox and there you will have Outh2 section, where you can generate access token. Keys are no longer supported.

You don't need to specify dropbox filesystem as default disk, you must only add dropbox disk to disks array in `config/filesystems.php`: https://github.com/mplodowski/dropboxadapter-plugin#configuration

Uploads and media files configuration is optional and not connected to Backup Manager plugin.

**Comment by MilosStanic** [comment]
Thank you very much for quick replies.
